### ISSUE TITLE
Segment and connection box definitions required for entire 50T graph

### ISSUE THREAD

**Issue by litghost** [SymbiFlow/symbiflow-arch-defs#1059]
The full 50T graph has more wire types (segments in VPR vocabulary), that need to be group for lookahead purposes.  The following segment and connection box reports can be generated from the final channels.db for the `50T`, or on a subset of the `50T` graph.

SQL for segment report:
```
SELECT tile_type.name, wire_in_tile.name, segment.name
FROM wire
INNER JOIN wire_in_tile ON wire_in_tile.pkey = wire.wire_in_tile_pkey
INNER JOIN tile_type ON tile_type.pkey = wire_in_tile.phy_tile_type_pkey
INNER JOIN node ON wire.node_pkey = node.pkey
INNER JOIN track ON track.pkey = node.track_pkey
INNER JOIN segment ON track.segment_pkey = segment.pkey
WHERE node.track_pkey IS NOT NULL;
```

Invoked like:
```
cd build
make file_xc7_archs_artix7_devices_xc7a50t-virt_channels.db
cd xc7/archs/artix7/devices/xc7a50t-virt
sqlite3 channels.db < segment_script.sql > segment_report_dups.txt
sort < segment_report_dups.txt | uniq > segment_report.txt
```

Where segment_script.sql is the context above.  This report takes ~6 minutes to generate on my machine, example output: [segment_report.txt](https://github.com/SymbiFlow/symbiflow-arch-defs/files/3704105/segment_report.txt)

@acomodi

**Comment by litghost** [comment]
Currently ~6000 wires are reported as unknown.  Some likely should've been marked OUTPINFEED like:
```
VFRAME|VFRAME_LOGIC_OUTS_B10|unknown
```
but further investigation is required.

Wires that likely need focus are the new clock network wires, examples:
```
CLK_BUFG_BOT_R|CLK_BUFG_BOT_R_CK_MUXED7|unknown
CLK_BUFG_BOT_R|CLK_BUFG_BOT_R_CK_MUXED8|unknown
CLK_BUFG_BOT_R|CLK_BUFG_BOT_R_CK_MUXED9|unknown
CLK_BUFG_BOT_R|CLK_BUFG_CK_GCLK0|unknown
CLK_BUFG_BOT_R|CLK_BUFG_CK_GCLK10|unknown
```

Another thing that needs to be watched for is clock network wires that became INPINFEED or OUTPINFEED.

**Comment by litghost** [comment]
Fixed in #1060
